src/main/java/com/hedvig/backoffice/graphql/types/FireDamageClaim.java

```diff
@@ -0,0 +1,49 @@
+package com.hedvig.backoffice.graphql.types;
+
+import com.hedvig.backoffice.services.claims.dto.ClaimData;
+import lombok.Value;
+import lombok.val;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Value
+public class FireDamageClaim {
```

Comment:
```suggestion
@UnionType
public class FireDamageClaim {
```

Also, remember to import it 🙂

Reply:
Correcto, gracias :cactus: